lib/models/device.coffee

```diff
@@ -1760,6 +1760,70 @@ getDeviceModel = (deps, opts) ->
 						uuid: uuid
 		.asCallback(callback)
 
+	###*
+	# @summary Set lock overriden
+	# @name setLockOverriden
+	# @public
+	# @function
+	# @memberof balena.models.device
+	#
+	# @param {String|Number} uuidOrId - device uuid (string) or id (number)
+	# @param {Boolean} shouldOverride - should override lock
+	# @returns {Promise}
+	#
+	# @example
+	# balena.models.device.setLockOverriden('7cf02a6', true);
+	#
+	# @example
+	# balena.models.device.setLockOverriden(123, true);
+	#
+	# @example
+	# balena.models.device.setLockOverriden('7cf02a6', true, function(error) {
+	# 	if (error) throw error;
+	# });
+	###
+	exports.setLockOverriden = (uuidOrId, shouldOverride, callback) ->
```

Comment:
I personally find methods like `enableLockOverride` and `disableLockOverride` more intuitive to use and more difficult to call with the wrong argument, so maybe we can name it like that? The `setLockOverriden` one can be private and stay pretty much the same (except maybe pass `'1'` and `'0'`) and be called by the two methods I mentioned above.